### PR TITLE
Update current node version in LA/Express

### PR DESCRIPTION
**Pull request by sgadiyar06**
Updated the current LTS and latest version of Node.js

#### Summary

Updating the LTS version and the latest version of Node

#### Motivation

Keeping it up to date

#### Supporting details

Current LTS and latest version are obtained from the node [website](https://nodejs.org/en/)

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

**Review by teoli2003 — CHANGES_REQUESTED** [review]
I think that such a sentence is unmaintainable. We should rephrase this paragraph to be date-independent. Mentioning ever-moving version number is not really useful, and almost always wrong.

**Comment by sgadiyar06** [comment]
Is it better to remove the version numbers altogether in this case?

File: files/en-us/learn/server-side/express_nodejs/introduction/index.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ While _Express_ itself is fairly minimalist, developers have created compatible
 
 ## Where did Node and Express come from?
 
-Node was initially released, for Linux only, in 2009. The NPM package manager was released in 2010, and native Windows support was added in 2012. At time of writing the current LTS release is Node v12.18.4 while the latest release is Node 14.13.0. This is a tiny snapshot of a rich history; delve into [Wikipedia](https://en.wikipedia.org/wiki/Node.js#History) if you want to know more.
+Node was initially released, for Linux only, in 2009. The NPM package manager was released in 2010, and native Windows support was added in 2012. At time of writing the current LTS release is Node v16.14.2 while the latest release is Node 17.9.0. This is a tiny snapshot of a rich history; delve into [Wikipedia](https://en.wikipedia.org/wiki/Node.js#History) if you want to know more.
```
Comment (teoli2003) [inline files/en-us/learn/server-side/express_nodejs/introduction/index.md:122]: I think this is better.

```suggestion
Node was initially released, for Linux only, in 2009. The NPM package manager was released in 2010, and native Windows support was added in 2012. Delve into [Wikipedia](https://en.wikipedia.org/wiki/Node.js#History) if you want to know more.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Review by teoli2003 — APPROVED** [review]
👍

**Comment by teoli2003** [comment]
Thanks a lot!

Looks good to me.

Congratulations upon your first merged commit here! Welcome aboard! :tada:

**Pull request merged**